pkg/asset/rhcos/image.go

```diff
@@ -67,6 +68,9 @@ func (i *Image) Generate(p asset.Parents) error {
 	case azure.Name:
 		//TODO(serbrech): change to right image once available.
 		osimage = "/resourceGroups/rhcos_images/providers/Microsoft.Compute/images/rhcostestimage"
+	case baremetal.Name:
+		// FIXME: baremetal
```

Comment:
can we track fixme in jira or github issue please rather than in code.

Reply:
I think this one is real, we depend on two images, the QEMU one for the bootstrap VM, and currently the OpenStack one for the baremetal hosts.

I have a WIP solution to this ref https://github.com/openshift-metal3/kni-installer/pull/100/commits/81ebb032f2a5c68ecfbd94129b732597d8baa2a7 so perhaps we can discus the best approach to that when we get to proposing that to openshift/installer